### DISCUSSION TITLE
add image for azzurro zcs inverter

### DISCUSSION THREAD

**Discussion by jolly12f** [slipx06/sunsynk-power-flow-card#317] [General]
Is it possible to add the ZCS azzurro
 inverter image for Italian users? 
![IMG_2497](https://github.com/slipx06/sunsynk-power-flow-card/assets/124673918/e95613e3-f6d2-4bd7-a880-e4f647c38526)

**Comment by slipx06** [comment]
Sure. Can you find a front on image, preferably PNG. All the images I see are angled

**Reply by jolly12f** [reply]
Ok this is ok? 
![IMG_2905](https://github.com/slipx06/sunsynk-power-flow-card/assets/124673918/44ef5d6d-24ad-47d3-a8e3-0ade459efb4a)

**Comment by slipx06** [comment]
@jolly12f this has been added to https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v4.25.0
